string action
string pose_name
string pose_type
